src/components/item/ItemLanguageSelector.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <b-container
+    fluid
+    class="border-bottom mt-3"
+  >
+    <b-container>
+      <b-row>
+        <b-col class="col-12 pb-3 d-inline-flex align-items-center flex-wrap">
+          <span>
+            <span class="icon-language pr-2" />
+            <i18n
+              path="multilingual.currentLanguage"
+              tag="span"
+              class="pr-1"
+              data-qa="translate item suggestion"
+            >
+              <span v-if="!translated && itemLanguage === selectedLocale.code">
+                a different language
+              </span>
+              <b-link
+                v-else
+                @click="translated ? toOriginal() : translate(selectedLocale.code)"
+              >
+                {{ languageToggle }}
+              </b-link>
+            </i18n>
+          </span>
+          <b-dropdown
+            :text="$t('multilingual.other')"
+            variant="link"
+            toggle-class="multilingual-dropdown"
+            toggle-tag="span"
+            no-flip
+          >
+            <b-dropdown-item
+              v-for="locale in availableLocales"
+              :key="locale.code"
+              class="multilingual-dropdown-item"
+              @click="translate(locale.code)"
+            >
+              {{ locale.name }}
+            </b-dropdown-item>
+          </b-dropdown>
+        </b-col>
+      </b-row>
+    </b-container>
+  </b-container>
+</template>
+
+<script>
+  import locales from '@/mixins/locales';
+
+  export default {
+    name: 'ItemLanguageSelector',
+    mixins: [
+      locales
+    ],
+    props: {
+      itemLanguage: {
+        type: String,
+        default: null
+      }
+    },
+    data() {
+      return {
+        translated: !!this.$route.query.metadataLang || false
```

Comment:
The translate profile is always going to be on, I think it's still useful to track translation status, but then let that signify that the item is currently _not_ shown in it's original metadata (edm) language. 
In which case this should be a computed property.